modules/nodes-nodes-resources-configuring-auto.adoc

```diff
@@ -73,7 +73,7 @@ spec:
 ----
 <1> Assign a name to CR.
 <2> Add the `autoSizingReserved` parameter set to `true` to allow {product-title} to automatically determine and allocate the `system-reserved` resources on the nodes associated with the specified label. To disable automatic allocation on those nodes, set this parameter to `false`.
-<3> Specify the label from the machine config pool.
+<3> Specify the label from the machine config pool. You can choose any desired labels for the machine config pool, such as `custom-kubelet=small-pods`, or the default label, `pools.operator.machineconfiguration.openshift.io/worker: ""`.
```

Comment:
I take a look at the customer cases conversions, the customer was confused that the label in the Tip is not used. I added the suggestions to point out the relation between the Prerequisites and the label here. 
Feel free to correct the wording.

```suggestion
<3> Specify the label from the machine config pool obtained from the Prerequisites step. You can choose any desired labels for the machine config pool, such as `custom-kubelet: small-pods` that is added using the above `oc label` Tip, or the default label, `pools.operator.machineconfiguration.openshift.io/worker: ""`.
```

Reply:
Thanks, @QiWang19 . I updated the callout based on your suggestion. I did not mention the `Tip` because I mentioned the `Prerequisites section`, which is a section that includes the tip. 

> Specify the label from the machine config pool that you configured in the _Prerequisites_ section. You can choose any desired labels for the machine config pool, such as `custom-kubelet= small-pods`, or the default label, `pools.operator.machineconfiguration.openshift.io/worker: ""`.

Reply:
Hi @bergerhoffer , is the above change OK with you?